xml/System/MathF.xml

```diff
@@ -1657,8 +1657,9 @@ The value of this field is 3.14159274.
   
 |Parameters|Return value|  
 |----------------|------------------|  
-|`x` or `y` = `NaN`.|`NaN`|  
-|`x` = Any value except `NaN`; `y` = 0.|1|  
+|`x` is any value; `y` = 0.|1|  
+|`x` = 1; `y` is any value.|1|  
+|`x` or `y` = `NaN`. (except for the above cases)|`NaN`|  
```

Comment:
```suggestion
|`x` or `y` = `NaN` (except if `y` = 0 or `x` = 1).|`NaN`|  
```

Reply:
We might want to do something similar to what @arturek did [here for Math](https://github.com/dotnet/dotnet-api-docs/commit/7db6ac602d5c6bc09b1428a8868385662571a7e5).

Reply:
> We might want to do something similar to what @arturek did [here for Math](https://github.com/dotnet/dotnet-api-docs/commit/7db6ac602d5c6bc09b1428a8868385662571a7e5).

Committed the suggestion for now but I agree on that.